content/post/074-explore-the-files-api-on-protoschool.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2019-10-29
+url: 2019-11-04-explore-the-files-api-on-protoschool
+title: Explore the Files API on ProtoSchool
+author: Teri Chadbourne
+---
+
+**TODO: FIX PUBLICATION DATE once finish previewing/editing**
+
+Today we're excited to announce the launch of a [brand new ProtoSchool tutorial exploring the **Regular Files API**](https://proto.school/#/regular-files-api), built by the awesome [Gil Domingues](https://github.com/dominguesgm). 🎉
+
+This new tutorial explores the methods at the top-level of js-ipfs (`add`, `get`, `cat`, etc.) that are custom-built for working with files. We refer to these methods as the Regular Files API to distinguish them from the Mutable File System (MFS), which mimics a more traditional file system interface with methods like `files.cp` and `files.mkdir`, hiding the complexity of immutable content addressing in IPFS. Check out the [first lesson](https://proto.school/#/regular-files-api/01) of the new tutorial for more on the differences between these two approaches, as well as some important distinctions between the Files API and the DAG API.
```

Comment:
```suggestion
This new tutorial explores the methods at the top-level of js-ipfs (`add`, `get`, `cat`, etc.) that are custom-built for working with files. We refer to these methods as the "Regular Files API" to distinguish them from the Mutable File System (MFS), which mimics a more traditional file system interface with methods like `files.cp` and `files.mkdir`, hiding the complexity of immutable content addressing in IPFS. Check out the [first lesson](https://proto.school/#/regular-files-api/01) of the new tutorial for more on the differences between these two approaches, as well as some important distinctions between the Files API and the DAG API.
```

Which one is which? is the Regular Files API the one that mimics traditional file system interfaces, or the MFS one? Ending the sentence and starting with "_____ mimics a more traditional..." would help avoid any potential confusion